### ISSUE TITLE
Link is missing a link role [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#312]
The “Find a site near you” link on the My Health Log Results C (“Not feeling well”) screen is not conveyed to TalkBack users as a link.

![health ResultC - NRV - Find a site near you](https://user-images.githubusercontent.com/7034772/92155225-05e6be80-edf5-11ea-870b-74f6ff4daabd.png)

**Remediation Guidance**
Use the `accessibilityRole` attribute on the interactive element (`TouchableOpacity`, `Pressable`, etc.) to convey a `link` role to TalkBack users.
```javascript
<TouchableOpacity onPress={…} accessibilityRole='link'>
  <Text>Find a site near you</Text>
</TouchableOpacity>
```

**Comment by AlanSl** [comment]
Should be fixed by https://github.com/project-vagabond/covid-green-app/pull/328 in the next build

**Comment by jonnyjames** [comment]
This issue is now fixed. Closing this issue.
